ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -1034,6 +1034,12 @@ nnet_t* define_nets_with_driver(ast_node_t* var_declare, char* instance_name_pre
 
             if (var_declare->types.variable.initial_value) {
                 new_net->initial_value = (init_value_e)var_declare->types.variable.initial_value->get_bit_from_lsb(i);
+
```

Comment:
> * just attach to unconn to retain initial behavior since some circuit might need it

@jeanlego If I understood your intention correctly, this code should be working properly to avoid undriven outputs having initial value. However, I think we should take care of other pins that don't have an initial value since they are still undriven. Also, in this way, the netlist clean up will find that these outputs are connected to unconn so it won't remove them however still there are not any latches appearing in output blif file.

Reply:
> primary outputs are dropped and should be connected to buf connected to unconn to be explicit about the behavior and retain the ports the way it is expected.

I think with the ELSE expression section I addressed this issue. Also, the abovementioned too.